mon-pix/translations/fr.json

```
@@ -1,208 +1,216 @@
+          "label": "'<abbr title=\"obligatoire\">'*'</abbr>' J'accepte les '<a href=\"{cguUrl}\" class=\"link\" target=\"_blank\">'conditions d'utilisation de Pix'</a>'"
         },
-        "improvingText": "Tentez d'obtenir le niveau supérieur !"
-      },
-      "reset": {
-        "label": "Remettre à zéro",
-        "description": "Remise à zéro disponible dans { daysBeforeReset, plural, =0 {0 jour.} =1 {1 jour.} other {# jours.} }",
-        "modal": {
-          "title": "Remise à zéro de la compétence",
-          "important-message": "Vos { earnedPix } Pix vont être supprimés.",
-          "important-message-above-level-one": "Votre niveau { level } et vos { earnedPix } Pix vont être supprimés.",
-          "warning": "Attention : si vous avez un parcours d’évaluation en cours, certaines questions pourront vous être reposées."
+        "email": {
+          "error": "Votre adresse e-mail n’est pas valide.",
+          "help": "(ex: nom@exemple.fr)",
+          "label": "Adresse e-mail"
+        },
+        "firstname": {
+          "error": "Votre prénom n’est pas renseigné.",
+          "label": "Prénom"
+        },
+        "lastname": {
+          "error": "Votre nom n’est pas renseigné.",
+          "label": "Nom"
+        },
+        "password": {
+          "error": "Votre mot de passe doit contenir 8 caractères au minimum et comporter au moins une majuscule, une minuscule et un chiffre.",
+          "help": "(8 caractères minimum, dont une majuscule, une minuscule et un chiffre)",
+          "label": "Mot de passe"
         }
       },
-      "start": {
-        "label": "Commencer"
+      "first-title": "Inscrivez-vous",
+      "instructions": "Les champs marqués de '<abbr title=\"obligatoire\" class=\"mandatory-mark\">'*'</abbr>' sont obligatoires",
+      "legal-information": "Les informations recueillies sur ce formulaire sont enregistrées dans un fichier informatisé par Pix pour permettre l'accès au service offert par Pix. Elles sont conservées pendant une durée de 5 ans maximum à compter du dernier accès au compte utilisateur, et archivées selon les délais de prescription légale (5 ans). Elles sont destinées à Pix et à ses prestataires techniques exclusivement. Conformément à la loi « informatique et libertés », vous pouvez exercer votre droit d'accès aux données vous concernant et les faire rectifier en envoyant un mail à dpd@pix.fr.",
+      "subtitle": {
+        "link": "connectez-vous à votre compte"
       }
-    },
-    "for-competence": "la compétence {competence}",
-    "link-to-profile": "Retour Profil",
-    "next-level-info": "{ remainingPixToNextLevel } pix avant le niveau { level }",
-    "tutorials": {
-      "title": "Cultivez vos compétences",
-      "description": "Voici une sélection de tutos qui pourront vous aider à gagner des Pix."
     }
+  },
+
+  "navigation": {
+    "homepage": "Page d'accueil",
+    "link-to-profile": "Retour Profil",
+    "pix": "Pix"
```

Ce sont tous des liens :astonished:
Soit on pourrait rajouter `link` à toutes ces clés, soit on reformule avec quelque chose comme:
```suggestion
  "navigation": {
    "homepage": "Page d'accueil",
    "back-to-profile": "Retour Profil",
    "pix": "Pix"
```